packages/terra-list/docs/MultiSelectSections.md

```diff
@@ -0,0 +1,28 @@
+# Multi Select List with sections
+
+The Terra Multi Select List is a structural component to arrange content within list/listitems with a configurable number of selectable items.
```

Comment:
We already have documentation of what a multi select list is. This should be a description of sections within a multi select list.